src/providers/dataProvider/defaults/users.ts

```diff
@@ -4,28 +4,32 @@ const users: User[] = [
     name: 'ian',
     password: 'admin',
     adminRights: true,
-    active: true
+    active: true,
+    roles: ['rco-power-user']
   },
   {
     id: 2,
     name: 'jason',
     password: 'user',
     adminRights: false,
-    active: true
+    active: true,
+    roles: ['rco-power-user']
```

Comment:
could you make `jason` the plain `rco-user` please. So we then have one active user for each of the roles.

Reply:
Done